spec/queries/get_content_collection_spec.rb

```diff
@@ -429,46 +429,46 @@
 
     context "search in" do
       context "with a single nested field" do
-        let(:search_in) { "details.body" }
+        let(:search_in) { [:'details.body'] }
```

Comment:
do we benefit from these being hashes? seems we have quite a lot of dot separated so might be easier as strings